### PR TITLE
[ Cloudpak 3.0 ] - Removal or CR, CRBs and webhook

### PR DESCRIPTION
**Pull request by rashmi43**
Removing creation of the following resources:
cluster role
cluster role bindings
webhook

Removing functions:
removeCsAnnotationFromCR
removeCsAnnotationFromCRB

https://github.ibm.com/IBMPrivateCloud/roadmap/issues/55707

https://github.ibm.com/IBMPrivateCloud/roadmap/issues/55692

### CODE REVIEW

File: common/scripts/.githooks/make_lint-all.sh [hunk 1]
```diff
@@ -15,4 +15,5 @@
 # limitations under the License.
 
 # Launches fmt and lint checks
-make lint-all
+#make lint-all
+make lint-scripts
```
Comment (rwhundley) [inline common/scripts/.githooks/make_lint-all.sh:19]: Why is this changing? I would think that we want to lint everything as a default, and the script itself is called `make_lint-all.sh`.

**Review by rwhundley — CHANGES_REQUESTED** [review]

File: pkg/controller/authentication/authentication_controller.go [hunk 2]
```diff
@@ -329,10 +309,8 @@ func (r *ReconcileAuthentication) Reconcile(contect context.Context, request rec
 // The clusterrole, clusterrolebinding and User resources created by Authentication
 func (r *ReconcileAuthentication) deleteExternalResources(instance *operatorv1alpha1.Authentication) error {
 
-	crMap := generateCRData()
-	crbMap := generateCRBData("dummy", "dummy")
 	userName := instance.Spec.Config.DefaultAdminUser
-	csCfgAnnotationName := res.GetCsConfigAnnotation(instance.Namespace)
+	//csCfgAnnotationName := res.GetCsConfigAnnotation(instance.Namespace)
```
[the remark this answers is not in the document]
Comment (rashmi43) [reply pkg/controller/authentication/authentication_controller.go:313]: updated it

**Review by rwhundley — APPROVED** [review]

**Review by rwhundley — APPROVED** [review]
/lgtm

**Pull request merged**